src/GMXAutomationBase.sol

```diff
@@ -33,7 +33,7 @@ contract GMXAutomationBase is Ownable2Step {
 
     // This should be empty after every transaction. It is filled and cleared each time checkLog is called.
     // mapping (uint256(feedId) => tokenAddress)
```

Comment:
Can you change this comment to reflect the new mapping (tokenAddress => uint256(feedId))